### PR TITLE
uiux(Settings): add slider to control audio volume

### LINKED ISSUE #843: add volume slider in profile -> sound section

### PR DESCRIPTION
**Pull request by PascalPrecht**
Screenshot 2020-09-17 at 12 17 47

Closes #843

### CODE REVIEW

**Comment by PascalPrecht** [comment]
Few things to note here:

- There's no design for pure sound settings just yet, so it's very rudimentary
- This configuration should probably go into the notifications screen
- The slider controls volume for all Audio elements, this means: sounds for sending/receiving messages as well as audio messages
- We probably want to add "advanced" settings to control each of these volumes individually (need designs for that)

Long story short: this is a quick win for users that are annoyed by the loud sounds

File: ui/app/AppLayouts/Profile/Sections/SoundsContainer.qml [hunk 1]
```diff
@@ -3,13 +3,18 @@ import QtQuick.Controls 2.13
 import QtQuick.Layouts 1.13
 import "../../../../imports"
 import "../../../../shared"
+import "../../../../shared/status"
 
 Item {
     id: soundsContainer
     width: 200
     height: 200
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/SoundsContainer.qml:11]: Can remove those 2

File: ui/app/AppLayouts/Profile/Sections/SoundsContainer.qml [hunk 2]
```diff
@@ -3,13 +3,18 @@ import QtQuick.Controls 2.13
...
     Layout.fillHeight: true
     Layout.fillWidth: true
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/Sections/SoundsContainer.qml:13]: Either the anchors below will throw warnings because this is a Layout, or you can remove those 2 lines as well

**Review by jrainville — COMMENTED** [review]
There is also ErrorSound.qml that uses an Audio element
